resources/web/server/simulation_server.py

```diff
@@ -139,11 +143,77 @@ def __del__(self):
         self.kill_webots()
         self.cleanup_webots_instance()
 
-    def prepare_webots_instance(self):
-        """Setup a local Webots project to be run by the client."""
-        appPath = config['projectsDir'] + '/' + self.app + '/'
+    def setup_project(self):
+        logging.info("Setup project " + str(self.url))
```

Comment:
This line makes the simulation_server.py crashing in case of local project because `self.url` is not defined

Reply:
This kind of log is actually not very useful. I will remove them.